src/components/Pagination/Pagination.tsx

```diff
@@ -10,14 +10,19 @@ import generateUUID from "../../helpers/generateUUID";
 export interface PaginationProps {
   /** Additional className. */
   className?: string;
+  /** The currentPage can be used to programatically force the selected page to change
+   * without the user explicitly requesting it – for example, if the user should be
+   * brought back to the first page of a set of results after a new search. */
+  currentPage?: number;
   /** The callback function that takes a page number and returns a string
    * to use for a link's `href` attribute. This is used when the current
    * page should refresh when navigating. */
   getPageHref?: undefined | ((pageNumber: number) => string);
   /** ID that other components can cross reference for accessibility purposes. */
   id?: string;
-  /** The current page selected. */
-  initialPage: number;
+  /** The initially selected page (default value is 1). If using with `getPageHref`,
+   * this value will reset with each new page refresh. */
```

Comment:
The second sentence depends on a consuming app's implementation, so it's okay to remove or it should be rephrased.